core/src/test/java/org/infinispan/filter/CompositeKeyValueFilterConverter.java

```diff
@@ -14,6 +19,7 @@
  * @author wburns
  * @since 7.0
  */
+@SerializeWith(CompositeKeyValueFilterConverter.Ext.class)
```

Comment:
> why using the annotation instead of creating an AdvancedExternalizer like in the other classes?

Actually, it's a test class so it's fine as is :)